Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM node:5.1.1
+FROM ubuntu
```

Comment:
what version? i noticed that one of our servers is running node 0.10 i think, so should i build this against node4?